src/components/step-sequence/step-sequence-item/step-sequence-item.spec.tsx

```diff
@@ -26,7 +32,61 @@ describe("StepSequenceItem", () => {
     hiddenCurrentLabel: "HiddenCurrent",
   };
 
-  describe("when complete", () => {
+  describe("when status is incomplete", () => {
+    it.each(orientations)(
+      "renders the correct styling when orientation is %s",
+      (orientation) => {
+        const wrapper = render({
+          ...defaultProps,
+          status: "incomplete",
+          orientation,
+        });
```

Comment:
```suggestion
      const wrapper = mount(
          <StepSequence orientation={orientation}>
              <StepSequenceItem status="incomplete" indicator="1">Name</StepSequenceItem>
          </StepSequence>
      );
```
I'm not sure if this test would work this way since `StepSequenceItem` does not accept an `orientation` prop and instead relies on context passed down from the parent `StepSequence`. Would it be worth doing the suggested, mocking the context or moving this test to `step-sequence.spec.tsx`?

Reply:
This test works as the `StepSequenceItem` component spreads `rest`; we actually only care about rendering the underlying `StyledStepSequenceItem` here but I decided to use existing render function to achieve it rather than writing a long form mount just for this. There's already snapshot tests in `step-sequence.spec.tsx` that capture the behaviour of the whole component in the different `orientation`s so this one is more focused on the styling of the underlying sc.